api/libproject/devstart.rb

```diff
@@ -1632,21 +1633,24 @@ def update_user_disabled_status(cmd_name, args)
   token = token.chomp
   header = "Authorization: Bearer #{token}"
   content_type = "Content-type: application/json"
-  payload = "{\"email\": \"#{op.opts.user}\", \"disabled\": \"#{op.opts.disabled}\"}"
+  payload = "{\"username\": \"#{op.opts.user}\", \"accountDisabledStatus\": {\"disabled\": \"#{op.opts.disabled}\"}}"
+  endpoint = "/v1/admin/users/updateAccount"
```

Comment:
Ruby is new for me. Did you move the endpoint into a variable for readability, or do you anticipate it changing?

Reply:
both - I did change it here :)